app/scripts/modules/kubernetes/src/v2/manifest/ManifestQos.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { get } from 'lodash';
+
+import { HoverablePopover } from '@spinnaker/core';
+import './manifestLabels.less';
+
+export interface IManifestQosProps {
+  manifest?: {
```

Comment:
Can you add a TODO (for me) with a link to this issue here: spinnaker/spinnaker#3239?